cmd/nodelink-controller/main.go

```diff
@@ -62,8 +62,6 @@ const (
 
 	// machineAnnotationKey is the annotation storing a link between a node and it's machine. Should match upstream cluster-api machine controller. (node.go)
 	machineAnnotationKey = "cluster.k8s.io/machine"
```

Comment:
Eventually we should reference this symbolically.

Waiting for https://github.com/kubernetes-sigs/cluster-api/pull/663 to merge.